### PR TITLE
Worked on the liquidity deposit and withdraw modal design

### PR DESCRIPTION
**Pull request by Sanjeev1308**

### CODE REVIEW

**Review by mohsenParvar — APPROVED** [review]

File: src/app/pages/Liquidity/components/modal/Withdraw.tsx [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+            <Grid item>
+              <Grid container justifyContent="space-between">
+                <Grid item>
+                  <Text variant="body2">
+                    {t(translations.LiquidityPage.Modal.MaxSlippage())}
+                  </Text>
+                </Grid>
+
+                <Grid item>
+                  <Text variant="body2">0.1%</Text>
```
Comment (saito-sv) [inline src/app/pages/Liquidity/components/modal/Withdraw.tsx:80]: lets address these hardcoded values.

**Review by saito-sv — APPROVED** [review]
LGTM

File: src/app/pages/Liquidity/components/modal/Withdraw.tsx (hunk 1 above)
Comment (Sanjeev1308) [reply src/app/pages/Liquidity/components/modal/Withdraw.tsx:80]: Worked on the design, will removed once we gets dynamic data from API/RPC call

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-liquidity-modals.dev-app.axial.exchange

**Review by bigwampa — CHANGES_REQUESTED** [review]
This can't be reviewed until the pools are connected. There is no way to currently reach the modal.

**Comment by saito-sv** [comment]
This PR is automatically being deployed to https://preview-task-sanjeev-liquidity-modals.dev-app.axial.exchange

**Comment by Sanjeev1308** [comment]
> This can't be reviewed until the pools are connected. There is no way to currently reach the modal.

1. Deposit modal: Go to the liquidity page > clicks on deposit button > again clicks on deposit button
2. Withdraw modal: Go to the liquidity page > clicks on Withdraw button > again clicks on Withdraw button

**Review by bigwampa — CHANGES_REQUESTED** [review]
Cancel button should close both modals
Screen Shot 2022-03-22 at 3 43 11 PM

Withdraw token should be set to "Combo" by default. Currently no selection by default.
Screen Shot 2022-03-22 at 3 44 28 PM

Withdraw Percentage should be 100% by default. Currently set to 50% by default
Screen Shot 2022-03-22 at 3 45 20 PM

**Review by bigwampa — APPROVED** [review]

**Pull request merged**